engine/flink/management/src/it/resources/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM flink:1.7.2-scala_2.11
+FROM flink:1.7.2-scala_${scala.binary.version}
```

Comment:
Maybe we should also support different flink version? Because we will support alsow flink 1.9: https://github.com/TouK/nussknacker/pull/247?

Reply:
In theory yes it would be great, but it's kind of tricky, because we'd have to release versions like:
nussknacker-api_1.9_2.11  :)
And keeping code base working with two versions of flink is more tricky than with two scala versions...
I think we'll have to rather have branch like flink_{older}_backports and release it when needed...